recipe.yaml

```diff
@@ -38,6 +38,7 @@ Manifests:
                 -e AWS_IOT_THING_NAME \
                 -e AWS_GG_NUCLEUS_DOMAIN_SOCKET_FILEPATH_FOR_COMPONENT \
                 -e SVCUID \
+                -e HTTP_PROXY \
```

Comment:
Please add a sentence to the README to document the HTTP Proxy behaviour and option.

Reply:
Ok. The HTTP_PROXY environment variable is automatically set by greengrass whenever a proxy is configured in the nucleus component configuration.